src/reducers/index.js

```diff
@@ -47,6 +47,8 @@ const initialState = {
   addressesFound: 0,
   // Quantity of transactions already loaded to give a feedback to the user
   transactionsFound: 0,
+  // Height of the network arrived from ws data
```

Comment:
I think we should explain better what height of the network means because there are at least two different definitions.

Reply:
Improved the comment explaining that we are getting the height of the best chain